src/main/resources/application.yml

```diff
@@ -0,0 +1,80 @@
+# All env variables that are used in one place
+# This is for deployment-specific values, which may be managed by other teams
+env:
+  db:
+    host: jdbc:postgresql://${DATABASE_HOSTNAME:127.0.0.1}:5432
+    init: ${INIT_DB:false}
+    stairway:
+      name: ${STAIRWAY_DATABASE_NAME:stairwaylib}
+      pass: ${STAIRWAY_DATABASE_USER_PASSWORD:stairwaypwd}
+      user: ${STAIRWAY_DATABASE_USER:stairwayuser}
+    ws:
+      name: ${STAIRWAY_DATABASE_NAME:wm}
+      pass: ${STAIRWAY_DATABASE_USER_PASSWORD:wmpwd}
+      user: ${STAIRWAY_DATABASE_USER:wmuser}
+  logging:
+    project-id: ${SERVICE_GOOGLE_PROJECT:broad-dsde-dev}
+    sa-path: ${CLOUD_TRACE_SA_PATH:rendered/service-account.json}
+    sampling-probability: ${SAMPLING_PROBABILITY:0}
+    trace-enabled: ${CLOUD_TRACE_ENABLED:false}
+  urls: # While we've traditionally thought of these as env specific and ok to hardcode, with kubernetes they may change
+    sam: ${SAM_ADDRESS:https://sam.dsde-dev.broadinstitute.org/}
+    terra-datarepo: ${TERRA_DATAREPO_URL:https://jade.datarepo-dev.broadinstitute.org}
+
+# Below here is non-deployment-specific
+
+logging.pattern.level: '%X{requestId} %5p'
+
+server:
+  compression:
+    enabled: true
+    mime-types: text/css,application/javascript
+  port: 8080
+
+spring:
+  application.name: workspace
+  cloud.gcp:
+    credentials.location: file:${env.logging.sa-path}
+    trace:
+      enabled: ${env.logging.trace-enabled}
+      project-id: ${env.logging.project-id}
+  resources:
+    cache.cachecontrol:
+      max-age: 0
+      must-revalidate: true
+    static-locations: classpath:/api/
+  sleuth.sampler.probability: ${env.logging.sampling-probability}
+
+workspace:
+  datarepo.instances:
+    terra: ${env.urls.terra-datarepo}
+    terra-alpha: https://data.alpha.envs-terra.bio
+    terra-dev: https://jade.datarepo-dev.broadinstitute.org
+    terra-prod: https://jade-terra.datarepo-prod.broadinstitute.org
+    terra-staging: https://data.staging.envs-terra.bio
+
+  job:
+    max-threads: 4
+    polling-interval-seconds: 1
+    resource-id: mc-terra-workspace-manager
+    timeout-seconds: 1800
+
+  sam.base-path: ${env.urls.sam}
+
+  stairway-database:
+    force-clean: true
+    migrate-upgrade: ${env.db.init}
+    password: ${env.db.stairway.pass}
+    uri: ${env.db.host}/${env.db.stairway.name}
+    username: ${env.db.stairway.user}
+
+  status-check:
+    frequency-ms: 60000
+    staleness-threshold-ms: 600000
+
+  workspace-database:
+    initialize-on-start: ${env.db.init}
+    password: ${env.db.ws.pass}
+    upgrade-on-start: true
```

Comment:
we may eventually want this to be a variable. No need to change as part of this PR.